app/src/main/java/org/commcare/dalvik/abha/ui/main/activity/TestActivity.kt

```diff
@@ -28,18 +28,18 @@ class TestActivity : AppCompatActivity() {
             startIntentA()
         }
 
-        HeaderInterceptor.API_KEY = "Token 01bed27f81885164999b2adc0e28b8ba8cb58eda"
```

Comment:
@akashkj since this token is already leaked wanted to check if we can expire this token on HQ ?

Reply:
yes ,I have already asked @akashkj to remove or expire the token